app/views/memberships/_request_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= simple_form_for [parent, resource], remote: true do |f| %>
   <%= f.hidden_field :type, value: 'request' %>
 
-  <%= f.submit t('membership.new.request'), class: 'btn btn-primary btn-block' %>
+  <%= f.submit t('membershipable.new.request'), class: 'btn btn-primary btn-block' %>
```

Comment:
let's revert, we're not creating a new "membershipable" object (contest/group) here